internal/sysfs/file_unix.go

```diff
@@ -0,0 +1,21 @@
+//go:build unix || darwin || linux
```

Comment:
nit: unix already includes darwin and linux, does it not?

Reply:
as long as unix works in go 1.18 sounds good to remove it. I forget when it was added.

Reply:
interesting I was overly specific because I recalled sometimes just using `unix` didn't work. I just tried and it _does_ work, but if I just rely on the naming convention `file_unix.go` then it won't

Reply:
I think it _doesn't_ work with go 1.18? 🤔 they're all timing out 
 https://github.com/tetratelabs/wazero/actions/runs/5265550045/jobs/9518327188

Reply:
reverted the change as apparently this won't work yet on 1.18

Reply:
You're right: https://tip.golang.org/doc/go1.19#go-unix
🤦